setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
ignore line lengths (let `black` take care of that) and another conflict with `black` over binary operator line break placement

Reply:
Why is there a `setup.cfg` _and_ `setup.py`? Is there any other place to put this information?

Reply:
I ended up needing the `setup.cfg` for the [flake8 configuration](https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations). I couldn't see where in `setup.py` I would put it.

Reply:
Seems like it's still going to be used per [PEP 518](https://peps.python.org/pep-0518/#sticking-with-setup-cfg) (even if there's not much enthusiasm in that PEP for it)

Reply:
And flake8 has an [issue outstanding](https://github.com/PyCQA/flake8/issues/234) to support `pyproject.toml`

Reply:
It seems like `tox.ini` and `.flake8` are bother alternatives to putting this in `setup.cfg`. Either seems preferable to defining both a `setup.py` and `setup.cfg`, since I have concerns about having both of those files in the same package

Reply:
👍